assets/scss/_code.scss

```diff
@@ -1,101 +1,84 @@
 // Code formatting.
 
 .td-content {
-	// Highlighted code.
```

Comment:
It's hard to follow what has changed in this file. Could you consider not reformatting it in this PR so that the diffs are more clearly seen?

Did you only change the lines for the c2c button?

Reply:
There's nothing new in this file. I deleted all the styles for the tooltip and prettified it.